prow/config.yaml

```diff
@@ -57,3 +57,31 @@ presubmits:
         volumeMounts:
         - name: shared-mem
           mountPath: /dev/shm
+
+  GoogleCloudPlatform/oss-test-infra:
+  - name: pull-prow-config-validate
+    branches:
+    - master
+    decorate: true
+    always_run: true
+    extra_refs:
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190220-615bdbe2e-master
+        command:
+        - runner.sh 
+        args:
+        - bash
+        - -C
+        - cd
+        - /home/prow/go/src/k8s.io/test-infra
+        - "&&"
+        - bazel
+        - run
+        - prow/cmd/checkconfig
+        - --
+        - --config-path=/home/prow/go/src/github.com/GoogleCloudPlatform/oss-test-infra/prow/config.yaml
```

Comment:
/shrug

Reply:
`--config-path=prow/config.yaml` should be sufficient?

Reply:
that will validate the k8s.io/test-infra config I think... (I have a cd above)

Reply:
(let me try to mkpj)